docs/k8s.md

```diff
@@ -0,0 +1,79 @@
+# Konveyor Forklift k8s Installation Instructions
+
+## Pre-requisites
+
+- **Kubernetes cluster or Minikube v1.17+**
+- **OLM**
```

Comment:
Can you expand the OLM acronym? At least once at the top of the page, so people know what it stands for.

Reply:
Enhanced docs with more details , let me know...